### DISCUSSION TITLE
Genes in FeaturePlot from integration_introduction

### DISCUSSION THREAD

**Discussion by Chrisdoan9** [satijalab/seurat#6203] [Q&A]
Hello everyone,

Would anyone please help me to find where we can get the genes to put in FeaturePlot? I put the link to the article below. Thank you so much!
[FeaturePlot](https://satijalab.org/seurat/reference/FeaturePlot.html)(immune.combined, features = [c](https://rdrr.io/r/base/c.html)("**CD3D", "SELL", "CREM", "CD8A", "GNLY", "CD79A", "FCGR3A",
    "CCL2", "PPBP**"), min.cutoff = "q9")
[https://satijalab.org/seurat/articles/integration_introduction.html]

**Comment by samuel-marsh** [comment]
Hi,

I’m not sure what you mean by where to find the genes.  Your code posted above looks correct.  Could you clarify what you mean?

Best,
Sam

**Reply by Chrisdoan9** [reply]
Hi Sam,

I mean how we can figure out these genes: CD3D", "SELL", "CREM", "CD8A", "GNLY", "CD79A", "FCGR3A",
"CCL2", "PPBP" to put into the FeaturePlot(). Why not other genes?

Best,
Chris D

**Reply by samuel-marsh** [reply]
Got it.

The choice is totally up to you.  You can plot any gene in your dataset.  In this case in the vignette they have identified cluster markers and then are plotting those to visualization expression.  You follow this type of path or simply enter genes of interest, canonical marker genes from literature, etc.

Best,
Sam

**Reply by Chrisdoan9** [reply]
Thank Sam! Is that vignette in this case mean a description of how to do a task?

**Reply by samuel-marsh** [reply]
Yes, see the step right before the plotting code you posted.

**Reply by Chrisdoan9** [reply]
Hi Sam,

A follow-up question.

Did they identify cluster makers by using: FindMarker() and FindAllMarker() in this tutorial?
[https://satijalab.org/seurat/articles/pbmc3k_tutorial.html](https://satijalab.org/seurat/articles/pbmc3k_tutorial.html)
I was still confused because I could not find where these genes: "CD3D", "SELL", "CREM", "CD8A", "GNLY", "CD79A", "FCGR3A",
"CCL2", "PPBP". I looked at all the code and could not navigate to the section in the vingettes to get the exact these list of genes. FindMarker() outputs a list of genes and they are different from the list of genes used for FeaturePlot().

Thank you so much!
